lab_5_scrapper/scrapper.py

```diff
@@ -26,11 +95,64 @@ def _extract_config_content(self) -> ConfigDTO:
         Returns:
             ConfigDTO: Config values
         """
+        with open(self.path_to_config, 'r', encoding='utf-8') as file:
+            config_content = json.load(file)
+        return ConfigDTO(
+            seed_urls=config_content['seed_urls'],
+            total_articles_to_find_and_parse=config_content['total_articles_to_find_and_parse'],
+            headers=config_content['headers'],
+            encoding=config_content['encoding'],
+            timeout=config_content['timeout'],
+            should_verify_certificate=config_content['should_verify_certificate'],
+            headless_mode=config_content['headless_mode']
+        )
+
+
 
     def _validate_config_content(self) -> None:
         """
         Ensure configuration parameters are not corrupt.
         """
+        config = self._extract_config_content()
+
+        if not(
+                isinstance(config.seed_urls, list)
+                and all(re.match('https?://(www.)?', url) for url in config.seed_urls)
```

Comment:
more strict checks are needed - that these are seed urls for selected besite

Reply:
Changed it